### PR TITLE
turbogit 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by b4nst**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/turbogit.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    etc.install "config/tug.toml"
+
+    (bash_completion/"tug").write `#{bin}/tug completion bash`
+    (fish_completion/"tug.fish").write `#{bin}/tug completion fish`
+    (zsh_completion/"_tug").write `#{bin}/tug completion zsh`
+  end
+
+  test do
+    assert_match /Version:\s+#{version}/, shell_output("#{bin}/tug version")
```
Comment (dawidd6) [inline Formula/turbogit.rb:26]: Could this test be more sophisticated? I mean, test the functionality of the program more than just the version.

**Review by dawidd6 — CHANGES_REQUESTED** [review]
Repository currently has 0 stars.

**Comment by SMillerDev** [comment]
> Repository currently has 0 stars.

That should have failed the build.

**Comment by SMillerDev** [comment]
Hi, this seems like a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Pull request closed without merging**